app/views/steps/dwp/confirm_details/edit.html.erb

```diff
@@ -0,0 +1,65 @@
+<% title t('.page_title') %>
+<% step_header %>
+
+<div class="govuk-grid-row">
+   <div class="govuk-grid-column-two-thirds">
+      <%= govuk_error_summary(@form_object) %>
+   </div>
+</div>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full govuk-!-margin-bottom-5">
+    <h1 class="govuk-heading-xl"><%=t '.heading' %></h1>
+  </div>
+</div>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <table class="govuk-table app-dashboard-table">
```

Comment:
Sorry I just realised this is a table.

Tables are not very responsive in small devices, and for presenting pairs of key-value like this, we use instead definition lists ([summary list](https://design-system.service.gov.uk/components/summary-list/)). In this case, without the change links. There is an example in that page that is almost what we want here.

Let me know if you need help with this!